packages/adapter-static/README.md

```diff
@@ -71,6 +71,8 @@ export default {
 
 When operating in SPA mode, you can omit `config.kit.prerender.default` (or set it to `false`, its default value), and only pages that have the [`prerender`](https://kit.svelte.dev/docs/page-options#prerender) option set will be prerendered at build time.
 
+If Node.js throws errors when loading your pages during the build process and this is hard for you to work around, setting `config.kit.prerender.entries` to `[]` will prevent the build process from attempting to load any page or layout components. Do not set `config.kit.prerender.enabled` to `false` because that will also prevent the fallback page from being generated.
```

Comment:
Interesting, I just converted a project to SvelteKit yesterday, using the SPA mode, and I did not have this problem. Setting `prerender.enabled` to `false` worked for me, the final build folder contained the specified `index.html`

Reply:
As of this PR, the fallback `index.html` wouldn't be generated if `prerender.enabled` is set to `false`

Reply:
@dummdidumm Here's the timeline of events :)

- Before version 292, the fallback page would be created even if prerendering was disabled.
- 292 moved the prerender into the build process and as a side effect caused the fallback page to not be generated when `prerender.enabled: false`.
- I submitted a PR to change this which was merged into 304, but it caused problems for people who were doing things like instantiating database clients in the global scope from `hooks.js`, hence this PR.
- While investigating solutions, I discovered that setting `prerender.entries: []` would work for cases where Node.js will complain if it even tries to `import` the page/layout components.
- And so this PR, which reverts back to the 292 behavior while documenting what to do it that causes issues.

Reply:
ah yeah my bad, obviously this worked for me because the change wasnt reverted at that time, thanks for the clarification 😄